### PR TITLE
Add development scripts to facilitate packaging

### PR DESCRIPTION
**Pull request by kevinsung**

### CODE REVIEW

File: dev_tools/packaging/produce-package.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
```
Comment (babbush) [inline dev_tools/packaging/produce-package.sh]: This line shouldn't be here

File: dev_tools/packaging/publish-dev-package.sh [hunk 2]
```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
```
Comment (babbush) [inline dev_tools/packaging/publish-dev-package.sh]: delete

File: dev_tools/packaging/verify-published-package.sh [hunk 3]
```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
```
Comment (babbush) [inline dev_tools/packaging/verify-published-package.sh]: delete

File: dev_tools/packaging/produce-package.sh (hunk 1 above)
Comment (kevinsung) [reply dev_tools/packaging/produce-package.sh]: Why not? These are bash scripts so it's appropriate to start off with a shebang.

**Comment by kevinsung** [comment]
This is the analog of https://github.com/quantumlib/OpenFermion/pull/486 here.

**Pull request merged**